vcd/resource_vcd_vapp_vm.go

```diff
@@ -661,7 +661,7 @@ func resourceVcdVAppVmCreate(_ context.Context, d *schema.ResourceData, meta int
 	timeElapsed := time.Since(startTime)
 	util.Logger.Printf("[DEBUG] [VM create] finished VM creation in vApp [took %f seconds]", timeElapsed.Seconds())
 
-	return genericVcdVmRead(d, meta, "create")
+	return genericVcdVmRead(d, meta, "resource")
```

Comment:
Note for reviewers, I changed this also as `genericVcdVmRead` does nothing with neither "create" or "update" values, but it does with resource (removing from tfstate if it is not found)

cc: @Didainius  @lvirbalas  @adezxc @dataclouder

Reply:
I don't have enough context to grasp this change. Naturally, most important is that it keeps VM operations intact. Leaving for @dataclouder @Didainius @adezxc to review.

Reply:
ACK. I have rechecked. Good to go.